src/main/java/org/folio/qm/service/impl/CreationStatusServiceImpl.java

```diff
@@ -40,4 +40,10 @@ public boolean updateByJobExecutionId(UUID jobExecutionId, RecordCreationStatusU
         return true;
       }).orElse(false);
   }
+
+  @Override
+  @Transactional
```

Comment:
I think there is no need for the transaction because save is already atomic operation

Reply:
removed